docs/installation/server-config/security.rst

```diff
@@ -132,28 +133,130 @@ Configuring Authorization of KSQL with Kafka ACLs
 Kafka clusters can use ACLs to control access to resources. Such clusters require each client to authenticate as a particular user.
 To work with such clusters, KSQL must be configured to :ref:`authenticate with the Kafka cluster <config-security-ssl-sasl>`,
 and certain ACLs must be defined in the Kafka cluster to allow the user KSQL is authenticating as access to resources.
-The list of ACLs that must be defined depends on whether the KSQL cluster is configured for
-:ref:`interactive <config-security-ksql-acl-interactive>` or :ref:`non-interactive (headless) <config-security-ksql-acl-headless>`.
+The list of ACLs that must be defined depends on the version of the Kafka cluster.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Kafka Cluster v2.0 and above
```

Comment:
A general comment for this doc: we need a way of identifying apache kafka and CP Kafka separately.  ie. "Apache Kafka 2.0  and above, and Confluent Platform Kafka 5.0 and above support wildcard acls", etc.

The doc only mentions AK, which is a bit incongruous because this is the documentation for the confluent platform.

If we were to pick one flavor to document, we would rather just document the Confluent Platform version rather than the AK version.

Reply:
Allso, keep in mind that you can also use text-string replace for certain product names, e.g. |cp| Confluent Platform, or |c3-short| for Control Center. For a complete list, see https://github.com/confluentinc/docs/blob/master/conf.py#L231.

Reply:
+1 to better distinction between AK and CP versions.